composer.json

```
@@ -16,7 +16,8 @@
   ],
   "require": {
     "php": ">=7.2.10",
-    "ext-curl": "*"
+    "ext-curl": "*",
+    "guzzlehttp/guzzle": "^6.5"
```

in tao we use ::  "guzzlehttp/guzzle": "~6.0" maybe it''s better to harmonize them wdyt?

---

This should not cause any conflict with tao. But we can move this to `~6.0` 
```suggestion
    "guzzlehttp/guzzle": "~6.0"
```